CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+cmake_minimum_required(VERSION 3.3.2)
+project(mlboard 
+        LANGUAGES C CXX)
+
+# Set required C++ standard to C++11.
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+
+ADD_SUBDIRECTORY(include/proto)
```

Comment:
You've added proto as a subdir, but not filewriter. How do you intend to use/organise filewriter; is it just part of mlboard, or do you intend for it to be a separate library?

Reply:
I intend it to be part of mlboard , every header file gets copied to include directory along with proto.pb.h and then user csn include the file <mlboard.hpp> 
Which will in turn have the file writer and everything imported

Reply:
Just like ensmallen.hpp which imports multiple files of ensmalle_bits folder

Reply:
Ok, can you explain more about your issue regarding installation; what do you want to happen, and what exactly is happening instead?

Reply:
I wanted it to copy all my file in include/ so that any one can use #include<mlboard.hpp> and then can run the file

As of now it does copy the proto folder in include/ but all other folder gets ignored